src/convertToObject.js

```diff
@@ -1,12 +1,19 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .split(';')
+    .filter(line => line.trim())
```

Comment:
[CODE KNOWLEDGE] - The filter method is not working as expected. The filter method checks if the line is not empty, but since the split method returns an array of strings, all strings, even the empty ones, are truthy. You should check if the trimmed line is not empty.